usaspending_api/etl/subaward_etl.py

```diff
@@ -12,7 +12,7 @@
 
 # These queries are directly from the broker
 D1_FILE_F_QUERY = """
-SELECT *
+SELECT DISTINCT ON (subcontract_num, fsrs_procurement.contract_number, fsrs_procurement.idv_reference_number) *
```

Comment:
Yes, I chatted with the broker folks - duplicates are largely due to how the join is performed, so this just removes the handful of cases where it duplicates rows.